src/components/users/TickForm.tsx

```diff
@@ -91,7 +92,7 @@ export default function TickForm ({ open, setOpen, setTicks, ticks, isTicked, cl
       userId: session.data?.user.metadata.uuid,
       style: style.name,
       attemptType: attemptType.name,
-      dateClimbed: dateClimbed,
+      dateClimbed: new Date(Date.parse(`${dateClimbed}T00:00:00`)), // Date.parse without timezone converts dateClimbed into local timezone.
```

Comment:
The backend accepts anything that `new Date(...)` will accept, so we could pass in a number. However, the `Yup.Date()` validation is a bit easier to read if these are dates.